spanner/batchwriter/batchwriter.go

```diff
@@ -15,7 +15,7 @@
 // Package spanner provides high-level abstractions for working with
 // Cloud Spanner that are not available from the core Cloud Spanner
 // libraries.
-package spanner
+package batchwriter
```

Comment:
If we need this package, we should probably name it "writer" so we can put other writers we might build later in the same package